spatialmath/sphere.go

```diff
@@ -116,6 +121,9 @@ func (s *sphere) EncompassedBy(g Geometry) (bool, error) {
 	if other, ok := g.(*sphere); ok {
 		return sphereInSphere(s, other), nil
 	}
+	if other, ok := g.(*capsule); ok {
+		return sphereInCapsule(s, other), nil
```

Comment:
I think you will also need to add checks like this to the point.go file

Reply:
Ack, good catch